src/Infrastructure/Adapter/PostGisRoadLineSectionMaker.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Adapter;
+
+use App\Application\Exception\GeocodingFailureException;
+use App\Application\GeocoderInterface;
+use App\Application\GeometryServiceInterface;
+use App\Application\RoadLine;
+use App\Application\RoadLineSectionMakerInterface;
+use App\Domain\Geography\HouseNumber;
+
+final class PostGisRoadLineSectionMaker implements RoadLineSectionMakerInterface
+{
+    public function __construct(
+        private GeocoderInterface $geocoder,
+        private GeometryServiceInterface $geometryService,
+    ) {
+    }
+
+    public function computeRoadLineSection(
+        RoadLine $roadLine,
+        string|null $fromHouseNumber,
```

Comment:
```suggestion
        ?string $fromHouseNumber,
```
Pareil pour les suivants